anycloud/cli/src/logger/mod.rs

```diff
@@ -18,6 +18,7 @@ pub enum ErrorType {
   ScaleFailed = 120,
   PostFailed = 121,
   RunAgzFailed = 122,
+  DuplicateDnsPrivateIp = 123,
```

Comment:
we need to duplicate this in `deploy`. hmm maybe there is a way to get rid of this interop

Reply:
`deploy` could have a build script to extract this definition from the Alan repo, or we could potentially have some sort of common definition (likely still in this repo, but easier than parsing Rust code and getting the relevant data).

Reply:
Yeah, probably just a JSON file here

Reply:
> Yeah, probably just a JSON file here

The issue is turning that JSON file into an enum in Rust will also be difficult.